### ISSUE TITLE
the ETH value is wrong...can you fix it?

### ISSUE THREAD

**Issue by HibouLemaitre** [ProbablyBen/ETHComboBot#1]
the ETH value is wrong...can you fix it?

I will post proofs of the error bellow... (after i upload the files online)

**Comment by ProbablyBen** [comment]
Hi,

Please upload a screenshot of what is wrong and steps to reproduce this issue.

**Comment by HibouLemaitre** [comment]
ok, i tried again ...please watch the value "ETH Balance" from BOT and compare it with the Balance at site from "screencapture..." file!
In the BOT values is stated >>> ETH Balance: 6268698
AND IN ETH COMBO account is stated >>> 0,06268698 ???

So what is wrong with your bot program?

I followed all steps to install the bot...made the right account exchanges...

can you clarify?
![screencapture-ethcombo-home-php-2018-03-26-18_40_55](https://user-images.githubusercontent.com/5556931/37923190-6f30db52-3126-11e8-9e87-78f4a028695c.png)
![ethcombo_012](https://user-images.githubusercontent.com/5556931/37923230-8d05cf70-3126-11e8-8c6d-8db5af51d895.jpg)

**Comment by HibouLemaitre** [comment]
another question...is this site paying?i see lots of persons telling that it is not!

**Comment by ProbablyBen** [comment]
I don't believe the site actually pays out. I learned this a few days after I wrote this program. I was unable to reproduce your issue. It may have to do something with the locale of your system.
